client/src/main/proto/app/cash/backfila/service.proto

```diff
@@ -49,6 +49,13 @@ message ConfigureServiceRequest {
   optional string connector_extra_data = 3;
 
   optional string slack_channel = 4;
+
+  // A service can have multiple variants, each registered independently with Backfila.
+  // Variants have their own set of runs, use different connector configs, etc but are grouped and still identify as the same service.
+  //
+  // As an example use case, a service could register different variants for a production and sandbox deployment.
```

Comment:
question, why not run a separate backfila instance in the sandbox deployment and register all the services there if this is your goal?

Reply:
Yeah my sense is that this problem will get worse here over time. That people will want different types beyond just sandbox.